### ISSUE TITLE
staking page kills Firefox

### ISSUE THREAD

**Issue by xlc** [polkadot-js/apps#8758]
I always have polkadot js apps as a pinned page with Firefox. However if I leave it on the Polkadot staking page, the tab will completely frozen after few hours. I can't close it when I get back to it. The only thing I can do is completely kill Firefox. Stay on other pages are fine.

I guess this is more or less Firefox bug but surely it is something on the staking page trigger this. Mostly likely a leak of some sort.

**Comment by jacogr** [comment]
In all honesty, the staking page is a dog when it comes to performance. There are no "quick fixes" here. However in the background (and as time permits), I'm re-looking at the structure.

So as of now, while there are certainly issues, there is nothing specifically that can and will be done on this issue as logged. However there is hope to make it less "of a dog" going forward.

**Comment by jacogr** [comment]
Closing. Not because it is not important, but just because as suggested, there is nothing really that can be done in the current structure.

However... there is work ongoing elsewhere that will address these types of issues (See https://github.com/polkadot-js/apps/pull/8822 - it will be quite some time until we swap over, the effort there gets priority as time permits - the linked PR just pushes the local up)

TL;DR Despite closing there is medium-term hope.
